docs/source/cpp/compute.rst

```diff
@@ -274,6 +274,18 @@ an ``Invalid`` :class:`Status` when overflow is detected.
 +--------------------------+------------+--------------------+---------------------+
 | divide_checked           | Binary     | Numeric            | Numeric (1)         |
 +--------------------------+------------+--------------------+---------------------+
+| ln                       | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
+| ln_checked               | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
+| log10                    | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
+| log10_checked            | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
+| log2                     | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
+| log2_checked             | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
```

Comment:
I would suggest to add a note stating that these functions return `float64` value for integral inputs and same type as input for floating-point inputs.